pkg/virt-operator/resource/generate/components/webhooks.go

```diff
@@ -273,9 +273,9 @@ func NewVirtAPIMutatingWebhookConfiguration(installNamespace string) *admissionr
 						admissionregistrationv1.Create,
 					},
 					Rule: admissionregistrationv1.Rule{
-						APIGroups:   []string{clone.GroupName},
-						APIVersions: clone.ApiSupportedWebhookVersions,
-						Resources:   []string{clone.ResourceVMClonePlural},
+						APIGroups:   []string{clonebase.GroupName},
+						APIVersions: clonebase.ApiSupportedWebhookVersions,
```

Comment:
Shouldn’t we also support fetching clones created before the upgrade? I’m not sure exactly how it works, but what if a clone was started before the upgrade and didn’t complete until after the upgrade finishes? Shouldn’t we monitor both the old and new versions for few versions?

Reply:
I wonder if we should support this, given that it’s currently in Alpha. It also seems like this concern is already [covered](https://github.com/kubevirt/kubevirt/pull/11955#issuecomment-2128668915).

Reply:
Thanks @Barakmor1!
This code does not relate to alpha/beta AFAICT. I've just created the `clonebase` alias for the import of the `"kubevirt.io/api/clone"` package.  So IIUC they're both still supported :+1:

Reply:
I've also added a line regarding this in the commit message. Thanks!